clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -845,9 +843,9 @@ else if (value instanceof Class)
         private final Number min;
         private final Number max;
 
-        private Range(Number min, Number max) {
+        private Range(Number min, Number maxInclusive) {
```

Comment:
Aren't both bounds inclusive? The inconsistency is a little annoying and this is a private constructor anyway. I think mentioning this in the javadoc below is good enough.

Reply:
added java doc comments and changed the parameter name to `max`